corehq/apps/users/admin.py

```diff
@@ -15,3 +17,12 @@ class DDPartialDigestAdmin(admin.ModelAdmin):
 
 admin.site.register(UserNonce, DDUserNonceAdmin)
 admin.site.register(PartialDigest, DDPartialDigestAdmin)
+
+
+class CustomUserAdmin(UserAdmin):
+    def has_add_permission(self, request):
+        return False
+
```

Comment:
I think I still need some context on what this is doing. Where would it get called? And does `request` need to be passed in if it is unused?

Reply:
got it, thanks